fruit-app/README.md

```diff
@@ -2,9 +2,22 @@
 
 Very basic CRUD application for fruit
 
-## Running locally
+## Local Development Setup
 
 - Install dependencies, `npm install`
 - Run the server, `npm run`
```

Comment:
Do you mind updating this command to `npm start`?

Reply:
Yup, updated to `npm start`

Reply:
:+1: